.flake8

```diff
@@ -1,26 +1,32 @@
-# Some PEP8 deviations are considered irrelevant to stub files:
-#       E301 expected 1 blank line
-#       E302 expected 2 blank lines
-#       E305 expected 2 blank lines
-#       E701 multiple statements on one line (colon)
+# The following rules are incompatible with or enforced by black:
+#       E203 whitespace before ':' -- scripts only
+#       E301 expected 1 blank line -- stubs only
+#       E302 expected 2 blank lines -- stubs only
+#       E305 expected 2 blank lines -- stubs only
+#       E501 line too long
+
+# Some rules are considered irrelevant to stub files:
+#       E701 multiple statements on one line (colon) -- disallows "..." on the same line
+#       F401 imported but unused -- does not recognize re-exports
+#            https://github.com/PyCQA/pyflakes/issues/474
+#       F822 undefined name in __all__ -- flake8 does not recognize 'foo: Any'
+#            https://github.com/PyCQA/pyflakes/issues/533
+
+# Rules that are out of the control of stub authors:
 #       E741 ambiguous variable name
-#       F401 imported but unused
 #       F403 import *' used; unable to detect undefined names
 #       F405 defined from star imports
-#       F822 undefined name in __all__
-
-# Nice-to-haves ignored for now
-#       E501 line too long
 
 [flake8]
 per-file-ignores =
-  *.py: E203, W503
+  *.py: E203, E501
   *.pyi: E301, E302, E305, E501, E701, E741, F401, F403, F405, F822
   # Since typing.pyi defines "overload" this is not recognized by flake8 as typing.overload.
   # Unfortunately, flake8 does not allow to "noqa" just a specific error inside the file itself.
+  # https://github.com/PyCQA/flake8/issues/1079
```

Comment:
This seems to be no longer true. The linked issue is closed.

Reply:
I tested it, it's still not possible. I believe the issue was closed, because the maintainers have no intention of implementing this.